### ISSUE TITLE
Need to change Firefox link, not in standard nav editor in Wagtail

### ISSUE THREAD

**Issue by valgrimm** [mdn/developer-portal#526]
See Editable nav items: https://drive.google.com/open?id=1CGwh35Ro-gltRBQPmp2eOTSMDAQQ-2cN
See Part of menu that needs to be changed:  https://drive.google.com/open?id=1iaqRgRLEZEeOJIb6AW3Yelo2YiVlkHAL

**Comment by stevejalim** [comment]
@valgrimm Can you clarify what the change is that needs to happen, please?

**Comment by valgrimm** [comment]
Yes, hadn't gotten to this when creating. https://www.mozilla.org/firefox/new/ should work. Current link is locked to English language. That URL adjusts based on visitor settings

**Comment by valgrimm** [comment]
Kadir: Need clarification that link is correct

Steve: Need location in site files for relevant code if possible

**Comment by valgrimm** [comment]
Ali to provide URL, this request is from Feedback Excel

**Comment by atopal** [comment]
Unless Ali, corrects me, the link should go to https://www.mozilla.org/firefox/developer/
